app/views/questions/asset_sale_income/edit.html.erb

```diff
@@ -1,3 +1,3 @@
-<% content_for :form_question, t("views.questions.asset_sale_income.title", :year => current_intake.most_recent_filing_year, :count => current_intake.filer_count) %>
+<% content_for :form_question, t("views.questions.asset_sale_income.title", year: current_intake.most_recent_filing_year, :count => current_intake.filer_count) %>
```

Comment:
:D

Reply:
If you're working on removing [hash rockets](https://www.rubydoc.info/gems/rubocop/0.46.0/RuboCop/Cop/Style/HashSyntax) then consider also changing `:count => ...` to `count: ...` on this line, too.